libraries/AC_AttitudeControl/AC_AttitudeControl.cpp

```diff
@@ -1067,7 +1068,8 @@ float AC_AttitudeControl::get_althold_lean_angle_max_cd() const
 // Return roll rate step size in centidegrees/s that results in maximum output after 4 time steps
 float AC_AttitudeControl::max_rate_step_bf_roll()
 {
-    float alpha = MIN(get_rate_roll_pid().get_filt_E_alpha(), get_rate_roll_pid().get_filt_D_alpha());
+    float dt_average = AP::scheduler().get_filtered_loop_time();
```

Comment:
As a side note it is slightly annoying that AP_Scheduler::get_filtered_loop_time() doesn't have a comment above it (all other methods in the class do have comments) so it takes extra digging to ensure it is really providing the results in seconds.

Reply:
This use of the filtered_loop_time() seems to be the most significant change in AC_PosControl.

Reply:
This only impacts the autotune step size.